scripts/publishMaven.js

```diff
@@ -43,6 +46,27 @@ function main() {
     }
 }
 
+/**
+ * Task gpg: Sign artifacts with GPG.
+ * 
+ * Required binaries:
+ * - gpg
+ * 
+ * Required Environment Variables:
+ * - artifactFolder: folder containing *.jar/*.pom files.
+ * - GPGPASS: passphrase of GPG key.
+ */
+function pgpSign(configs, artifactFolder) {
```

Comment:
is here `pgp` a typo? Meant to be `gpg`?

Reply:
nice catch. yeah, typo.